WPMedia/fusion-news-theme-blocks
# PEN-676: [NVDA | Firefox] Article Landing Page > Author's Profile: Social media profile icons are read inappropriately via NVDA 2019 on Firefox.

**If you have not filled out the checklist below, the pr is not ready for review.**

## Description
Add 'aria-label' to author-bio social links

## Jira Ticket
- [PEN-676](https://arcpublishing.atlassian.net/browse/PEN-676)

## Acceptance Criteria
_copy from ticket_

## Test Steps

1. In `Fusion-News-Theme` repo, checkout `master` & `git pull`
2. After checking out this feature branch in `fusion-news-theme-blocks`, run `rm -rf ./node_modules && npx lerna clean && npm i && npx lerna bootstrap && cd blocks/header-nav-chain-block && npm i && cd ../..`
3. Run `npx fusion start -f -l @wpmedia/author-bio-block` in `Fusion-News-Theme`
4. We are planning on turning these accessibility tickets over to the auditing firm that reported the issues in the first place. We don't have access to these tools and most are either paid or Windows-only. For that reason, just take a look over the code to make sure it makes sense and check http://localhost/local/2020/01/14/tampa-reinvents-itself-to-show-off-its-shores/?_website=the-gazette to make sure that the `aria-label` attributes are now present on the social links in the author bio (see screenshot below). 

![image](https://user-images.githubusercontent.com/26662906/107421807-07bd5800-6ae0-11eb-9c28-c2eed24fa8d6.png)


## Effect Of Changes
### Before
No `aria-label` attributes were present on the social links in the author bio

### After
There are now `aria-label` attributes with descriptive content present on the social links in the author bio

## Dependencies or Side Effects
_Examples of dependencies or side effects are:_
- Additional settings that will be required in the blocks.json
- Changes to the custom fields which will require users to reconfigure features
- Update to css framework or SDK
- Dependency on another PR that needs to be merged first

## Author Checklist
_The author of the PR should fill out the following sections to ensure this PR is ready for review._
please explain why (so that we can fix it whenever it gets refactored).

## Reviewer Checklist 
_The reviewer of the PR should copy-paste this template into the review comments on review._
